src/seedu/addressbook/data/person/Name.java

```diff
@@ -40,7 +40,7 @@ public static boolean isValidName(String test) {
      * Retrieves a listing of every word in the name, in order.
      */
     public List<String> getWordsInName() {
-        return Arrays.asList(fullName.split("\\s+"));
+        return Arrays.asList(fullName.toUpperCase().split("\\s+"));
```

Comment:
toUpperCase() may be slower toLowerCase() but the time taken should be negligible